src/app/phast/losses/losses-help/energy-input-help/energy-input-help.component.html

```diff
@@ -18,15 +18,114 @@
+        should be corrected for cycle time to obtain per hour value.
+      </p>
+      <label><u><strong>Input</strong></u></label>
+      <p>
+        Units: <strong>lbs/hr</strong> or <strong>kg/hr</strong><br> Minimum Value: <strong>0</strong><br>
+      </p>
+    </div>
+  </div>
+
+  <div class="card" *ngIf="currentField == 'coalHeatingValue'">
+    <div class="card-header">
+      Coal Carbon Injection
+    </div>
+    <div class="card-block">
+      <label><u><strong>Description</strong></u></label>
+      <p>
+        Heating value for the coal or carbon injected.
+      </p>
+      <label><u><strong>Input</strong></u></label>
+      <p>
+        Units: <strong>Btu/lb</strong> or <strong>kg/lb</strong><br> Minimum Value: <strong>0</strong><br>
+      </p>
+    </div>
+  </div>
+
+  <div class="card" *ngIf="currentField == 'electrodeUse'">
+    <div class="card-header">
+      Electrode Use
+    </div>
+    <div class="card-block">
+      <label><u><strong>Description</strong></u></label>
+      <p>
+        Mass (weight) of electrode use on a per hour basis. Note that for batch furnaces the electrode use should be corrected for cycle time to obtain per hour value.
+      </p>
+      <label><u><strong>Input</strong></u></label>
+      <p>
+        Units: <strong>lbs/hr</strong> or <strong>kg/hr</strong><br> Minimum Value: <strong>0</strong><br>
       </p>
     </div>
   </div>
 
-  <!--remaining fields needed-->
+  <div class="card" *ngIf="currentField == 'electrodeHeatingValue'">
+    <div class="card-header">
+      Electrode Heating Value
+    </div>
+    <div class="card-block">
+      <label><u><strong>Description</strong></u></label>
+      <p>
+        Heating value for the electrode material - graphite.
+      </p>
+      <label><u><strong>Input</strong></u></label>
+      <p>
+        Units: <strong>Btu/lb</strong> or <strong>kg/lb</strong><br> Minimum Value: <strong>0</strong><br>
+      </p>
+    </div>
+  </div>
 
-</div>
\ No newline at end of file
+  <div class="card" *ngIf="currentField == 'otherFuels'">
+    <div class="card-header">
+      Other fuels
+    </div>
+    <div class="card-block">
+      <label><u><strong>Description</strong></u></label>
+      <p>
+        Heat supplied from other sources, if any. Note that for batch furnaces the total weight should be corrected for cycle time to obtain per hour value.
+      </p>
+      <label><u><strong>Input</strong></u></label>
+      <p>
+        Units: <strong>mm Btu/hr</strong> or <strong>GJ/hr</strong><br> Minimum Value: <strong>0</strong><br>
+      </p>
+    </div>
+  </div>
+
+  <div class="card" *ngIf="currentField == 'electricityInput'">
+    <div class="card-header">
+      Electricity Input
+    </div>
+    <div class="card-block">
+      <label><u><strong>Description</strong></u></label>
+      <p>
+        Electricity Input into system Note that for batch furnaces the electricity input should be corrected for cycle time to obtain per hour value.
+      </p>
```

Comment:
"Electricity input into system. Note" instead of "Electricity Input into system Note"